blue-note/templates/single.html

```diff
@@ -1,27 +1,32 @@
 <!-- wp:template-part {"slug":"header-large","tagName":"header"} /-->
 
 <!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80","right":"var:preset|spacing|50","left":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
-<main class="wp-block-group" style="padding-top:var(--wp--preset--spacing--80);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--80);padding-left:var(--wp--preset--spacing--50)"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
-<div class="wp-block-group alignwide"><!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"100px","left":"100px"}}}} -->
-<div class="wp-block-columns"><!-- wp:column {"width":"66.66%","style":{"spacing":{"blockGap":"var:preset|spacing|60"}}} -->
-<div class="wp-block-column" style="flex-basis:66.66%"><!-- wp:post-content {"style":{"typography":{"fontSize":"22px"}}} /-->
-
-<!-- wp:group {"style":{"typography":{"fontSize":"22px"},"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-<div class="wp-block-group" style="font-size:22px"><!-- wp:paragraph -->
-<p>Author: </p>
-<!-- /wp:paragraph -->
-
-<!-- wp:post-author {"showAvatar":false,"showBio":false} /-->
-
-<!-- wp:paragraph -->
-<p>/</p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph -->
-<p>Tags: </p>
-<!-- /wp:paragraph -->
-
-<!-- wp:post-terms {"term":"post_tag"} /--></div>
+<main class="wp-block-group" style="padding-top:var(--wp--preset--spacing--80);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--80);padding-left:var(--wp--preset--spacing--50)">
+<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide">
+<!-- wp:columns {"style":{"spacing":{"blockGap":{"left":"8.13rem"}}}} -->
+<div class="wp-block-columns"><!-- wp:column {"width":"66.66%","style":{"spacing":{"blockGap":"var:preset|spacing|80"}}} -->
+<div class="wp-block-column" style="flex-basis:66.66%">
+<!-- wp:post-content /-->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group">
+	<!-- wp:paragraph -->
+	<p>Author: </p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:post-author-name /-->
+	
+	<!-- wp:paragraph -->
```

Comment:
Can we use [this pattern](https://github.com/WordPress/community-themes/blob/trunk/blue-note/patterns/post-meta-nav.php) for this bit?